splice_machine/src/main/java/com/splicemachine/derby/stream/utils/ExternalTableUtils.java

```diff
@@ -234,7 +235,8 @@ public static void sortColumns(StructField[] schema, int[] partitionColumnMap) {
 
     public static boolean isEmptyDirectory(String location) throws Exception {
         String[] files = ImportUtils.getFileSystem(location).getExistingFiles(location, "*");
-        if ((files.length == 0) || (files.length == 1 && files[0].equals("_SUCCESS")))
+
+        if ((files.length == 0) || (files.length == 1 && "_SUCCESS".equals(truncateFileNameFromFullPath(files[0]))))
```

Comment:
Rather than `if (true) return true else return false` do `return (files.length == 0) || (files.length == 1 && "_SUCCESS".equals(truncateFileNameFromFullPath(files[0])))`

Reply:
Thanks for pointing it.